### PR TITLE
Fixed readme link issues

### PR DESCRIPTION
**Pull request by praveen-klaytn**
## Proposed changes

- Fixed readme issues

## Types of changes

Please put an x in the boxes related to your change.

- [X] Bugfix
- [ ] New feature or Enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [X] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-developer-sdk/blob/main/CONTRIBUTING.md) doc
- [X] I have signed the CLA by posting "I have read the CLA Document and I hereby sign the CLA" in the comment
- [X] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/oracles-starter-kit/README.md [hunk 1]
```diff
@@ -519,7 +519,7 @@ npx witnet-toolkit try-query --from-solidity ./contracts/witnet-requests/<contra
...
...zc3ndRJA+rPp3z/8WqXOUO9SrGtcQm4O1bJysvHTHtdW8OxzZb8OmC6mK/GkeUd28TJa2WKbVUlYN3GjGMFxjTMGDYWTPnaGnrTFEdue+GYEt7LrTKol7OSjZeTYF61aV8AAIDoGJD3SGW/Ue2WfEc0383Uckv5ZWsfBOw2N+NeZyqdVCTX9FPq2iVde/Yauo5gQ8nF7kZqzPOIaa/xZpmyl2fKCawuWoZBwHA1vJv3me0dxzZ7Y80DBskfsXr0YzKxYMp785qvoTe9JC/JPEjKZ1I7vX/kUYpxjgAAAGJ5DNAtRbXzSEXdHsl1t72L+eYv9fZbSRAwJyLS5dB3ZC1fRQrihTqITkXH8mxsF5jnrCESra2KZ3euZtXL755cQ2+nE0kBqsirgPursZ6yGNuIeKzp8npxkfyuaRrCQ70hjvavM5TCmTXUbiWN6s4R9RAjG7N86Io64SKeV+GNziQJAADuoD8GbNLuzVsltusxq33MVAq82NxkdUDf3+T7Whg2N6MUzUfxAFqqikkd8MhGql2Wo9upVa8La1ivhgQqyZtVwNVqxFNusnYeRKzGqicr9+sM4+RX4jREPWK9vFO7T/nsGlrttWOASN3mvdIozUyFDSQdvJwdenuHNgAAAGY7BviNJ8h7m3O7JeAfyc8Ph7fmvpG/9PF5S18AAPDP9vRjQP4hCDYigLM/o5dPhZfX7I/zj7Qc8dEDAACBJx4D0p95CjYiQC//pM+zPhqrHyUCAAAw7/FDQQAAAAA+FY4BAAAAwO1wDAAAAABuh2MAAAAAcDscAwAAAIDb4RgAAAAA3M7LLxwDAAAAgJt5+ev3XzkGAAAAALfy8vcfv3EMAAAAAG6FHwoCAAAAbof/RRgAAAC4HY4BAAAAwO1wDAAAAABuh2MAAAAAcDscAwAAAIDb4RgAAAAA3A7HAAAAAOB2OAYAAAAAt8MxAAAAALgdjgEAAADA7XAMAAAAAG6HYwAAAABwOxwDAAAAgNvhGAAAAADcDscAAAAA4HY4BgAAAAC3wzEAAAAAuB2OAQAAAMDtcAwAAAAAbodjAAAAAHA7HAMAAACA2+EYAAAAANwOxwAAAADgdjgGAAAAALfz8gvHAAAAAOBmXv76/VeOAQAAAMCtvPz9x28cAwAAAIBb4YeCAAAAgJv56af/Af1pgdRm+b5VAAAAAElFTkSuQmCC)
```
Comment (toniya-klaytn) [inline packages/oracles-starter-kit/README.md:522]: why is this imp ? can't we just upload the png file give the path ?

Comment (toniya-klaytn) [inline packages/oracles-starter-kit/README.md:522]: Is this the link to binary ? or the binary itself ?

Comment (praveen-klaytn) [reply packages/oracles-starter-kit/README.md:522]: This is base64 of the file

Comment (praveen-klaytn) [reply packages/oracles-starter-kit/README.md]: Replaced above base64 witnet with github image from main branch

**Review by toniya-klaytn — APPROVED** [review]

**Pull request merged**